oracles.asciidoc

```diff
@@ -11,6 +11,7 @@ However, many blockchain applications require access to extrinsic information. T
 * Capital markets data: pricing baskets of tokenized assets/securities
 * Benchmark reference data: incorporating interest rates into smart financial derivatives
 * Static/pseudo-static data: security identifiers, country codes, currency codes
+* Time and Interval data: Event triggers grounded in precise SI time measurement
```

Comment:
Interval --> interval
Event --> event

Perhaps introduce the SI acronym if not already done so